systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -878,6 +992,18 @@ void testKafkaMirrorMaker2ConnectorsStateAndOffsetManagement() throws JsonProces
      * while user Scram passwords, CA cluster and clients certificates are changed.
      */
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Validates rolling update of MM2 after changing SCRAM-SHA user secrets and certificates."),
+        steps = {
+            @Step(value = "Deploy clusters and users with SCRAM-SHA.", expected = "SCRAM users and clusters are ready."),
+            @Step(value = "Deploy MM2 with SCRAM and CA credentials.", expected = "MM2 mirrors messages."),
```

Comment:
```suggestion
            @Step(value = "Deploy MM2 with SCRAM-SHA and CA credentials.", expected = "MM2 mirrors messages."),
```

what are CA credentials?

Reply:
CA credentials => CA certificates.